cmd/omegaup-grader/frontend_handler_test.go

```diff
@@ -341,10 +357,52 @@ func TestBroadcastRun(t *testing.T) {
 		verdict       string
 		score         *big.Rat
 		expectedScore float64
+		scorePerGroup map[string]float64
 	}{
-		{scoreMode: "partial", verdict: "AC", score: big.NewRat(1, 1), expectedScore: 1.},
-		{scoreMode: "partial", verdict: "PA", score: big.NewRat(1, 2), expectedScore: 0.5},
-		{scoreMode: "all_or_nothing", verdict: "PA", score: big.NewRat(1, 2), expectedScore: 0.},
+		{
+			scoreMode:     "partial",
+			verdict:       "AC",
+			score:         big.NewRat(1, 1),
+			expectedScore: 1.,
+			scorePerGroup: map[string]float64{
```

Comment:
```suggestion
			scorePerGroup: map[string]any{
```

con esto ya debe pasar